app/views/greetings/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hello Rails React</title>
+</head>
+<body>
+  <div id="root"></div>
+  <script src="/packs/application.js"></script>
+  <script>
+    import React from 'react';
+    import { render } from 'react-dom';
+    import App from '../javascript/packs/App';
+
+    document.addEventListener('DOMContentLoaded', () => {
+      render(<App />, document.getElementById('root'));
+    });
+  </script>
+</body>
+</html>
```

Comment:
- [ ] This file does not need to have as much code as this, just `<div id="root"></div>` is enough and you will use the document.getElementBtId('root') on from the react side.